src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicStream.cs

```diff
@@ -997,6 +1063,86 @@ private void ThrowIfDisposed()
             }
         }
 
+        private static uint HandleConnectionClose(State state)
```

Comment:
```suggestion
        private static uint HandleEventConnectionClose(State state)
```
I'd love to have the info about it being "Event" handler. We have other "Handle" methods that deal with state changes and are not necessarily related to msquic events.

Reply:
But strictly speaking the actual event's name here is SHUTDOWN_COMPLETE... It would sounds a bit confusing to me to have HandleEventConnectionClose while there's no ConnectionClose event inside stream 🤔 But of course if you think it would bring more value to highlight this method is called as a result to (some) event, I will rename it.